extensions/oidc-client/deployment/src/test/java/io/quarkus/oidc/client/OidcClientCredentialsJwtTlsRegistryTest.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.oidc.client;
```

Comment:
Unfortunately this test will have to go

Reply:
But updating `integration-tests/oidc-mtls` would be interesting, as there we have HTTP server MTLS, so testing it alongside the OIDC MTLS authentication with the tls registry config would be good

Reply:
I thought `integration-tests/oidc` already is already testing mTLS between provider and Quarkus app. Maybe I read it wrong.

Reply:
Because `integration-tests/oidc-mtls` tests mTLS between app user requests, not provider and Quarkus. I will put in additional work if there is positive in there, I just tried to avoid adjustments in Keycloak resource side.

Reply:
I'll remove this test.

Reply:
@michalvavrik `integration-tests/oidc` does test it, you are right, I thought testing a situation where a TLS registry is used to configures both HTTP server and OIDC can be useful to verify, but I can definitely try it later, that is fine